Swift_Alert/Swift_Alert/ContentView.swift

```diff
@@ -8,14 +8,104 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constants {
+        static let showTitle = "Показать"
+        static let firstBlockTitle = "Пример Alert"
+        static let secondBlockTitle = "Пример Alert c 2 кнопками и логикой"
+        static let secondBlockTitleAfterTap = "Другой текст"
+        static let thirdBlockTitle = "Пример ActionSheet"
+        static let fourthBlockTitle = "Пример ActionSheet c кнопками и логикой"
+        static let fourthBlockTitleAfterTap = "Пример ActionSheet c кнопками и логикой"
+        static let changeTitle = "Изменить текст"
+        static let changeColorTitle = "Сменить цвет?"
+        static let transferMoneyTitle = "Перевести деньги"
+        static let yesTitle = "Да"
+        static let cancelTitle = "Отмена"
+        static let areYouSureTitle = "Вы уверены?"
+        static let answerTitle = "Выберите вариант ответа"
+        static let spacing: CGFloat = 25
+
+    }
+    
+    @State var testFirstView = false
+    @State var testSecondView = false
+    @State var forFirstAlertFlag = false
+    @State var forSecondAlertFlag = false
+    @State var testActionSheet = false
+    @State var testActionSheetWithButton = false
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        VStack(spacing: Constants.spacing, content: {
+            Text(Constants.firstBlockTitle)
+            showAlert()
+            if !forFirstAlertFlag {
+                Text(Constants.secondBlockTitle)
+            } else {
+                Text(Constants.secondBlockTitleAfterTap)
+            }
+            showAlertWith()
+            Text(Constants.thirdBlockTitle)
+            showActionSheet()
+            if !forSecondAlertFlag {
+                Text(Constants.fourthBlockTitle)
+            } else {
+                Text(Constants.fourthBlockTitleAfterTap)
+                    .foregroundColor(.red)
+            }
+            showActionSheetButton()
+        })
+    }
+    
+    fileprivate func showAlert() -> some View {
+        return Button {
```

Comment:
```suggestion
        Button {
```
если компилятор явно не требует, то return не пишем